src/Directory.Build.props

```diff
@@ -9,10 +9,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="$(MicrosoftAzureServicesAppAuthentication)" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="$(MicrosoftExtensionHttpVersion)" />
-    <PackageReference Include="Microsoft.Identity.Client" Version="$(MicrosoftIdentityClientVersion)" />
     <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="$(MicrosoftRestClientRuntimePackageVersion)" />
+    <PackageReference Include="Azure.Identity" Version="$(AzureIdentityPackageVersion)" />
```

Comment:
`Microsoft.Azure.Services.AppAuthentication` is out of date.

`Azure.Identity` depends on `Microsoft.Identity.Client`, which supports the conditional access.